spring-cloud-gateway-core/src/test/java/reactor/blockhound/integration/CustomBlockHoundIntegration.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.blockhound.integration;
+
+import reactor.blockhound.BlockHound;
+
+/**
+ * @author Tim Ysewyn
+ */
+public class CustomBlockHoundIntegration implements BlockHoundIntegration {
+
+	@Override
+	public void applyTo(BlockHound.Builder builder) {
+		try {
+			Class.forName("ch.qos.logback.classic.spi.PackagingDataCalculator");
```

Comment:
Do you really need these `Class.forName`s? Aren't you sure about their presence?